src/api/views.py

```diff
@@ -8,6 +9,15 @@
 from django.utils import timezone
 import time
 
+def liveness(request):
+    return JsonResponse({"status": "alive"})
+def rediness(request):
```

Comment:
There is a typo in the function name `rediness`. It should be `readiness` to match the endpoint defined in `urls.py`.